### ISSUE TITLE
How does the script work?

### ISSUE THREAD

**Issue by dany-eduard** [Nicolasopf/Tools#1]
Hey, bro, I saw your Twitter post about version 1. I can deduce that this script performs a customization in Emacs, but I'd like to see some images of how it looks and what other modifications it makes.
I appreciate your response.

**Comment by Nicolasopf** [comment]
Yes sure, here is an example. (I will upload btw this image at the
repository, thanks)

![image](https://user-images.githubusercontent.com/69660332/102292147-995e7e00-3f12-11eb-9934-a699c4255454.png)
![image](https://user-images.githubusercontent.com/69660332/102292174-a5e2d680-3f12-11eb-8993-84bae67f2d9f.png)

**Comment by Nicolasopf** [comment]
Also, Tools will add an script called "sgit" that will be a shortcut for
git add, git commit and git push, this script have an automation if you
have pep8, that is a checker for a style code, this will check if You are
trying to upload a .py script and will execute pep8 (if you have, btw I
will added it to the repo) If you have something wrong with pep8 (style
code checker) You will exit of the sgit command. Also, if your file is a
.py this script will add execute permissions to your file. You can use sgit
as:
sgit file_to_add "all your commit"

The other config file (as .emacs) is the .bashrc, It detects if you have
enabled full path in your terminal, If you terminal looks like this:
ubuntu@~/directory/another_directory/tests:
as
ubuntu@~/tests:
Will show in the terminal only the actual directory and not all the parent
directory.

The basics file will try to install emacs git python3 Other features like
exe, remove_packages, same_file will be not installed if You don't move
that files to some in $PATH , but You can use they if You are in the
current directory by ./file

Pports will be only installed if you accept to install hack tools (At this
time, It will install only nmap and script pport)
I will add more documentation today in the repository, thanks!

>
